### PR TITLE
Jcaccamo/flyover ar

### PR DESCRIPTION
**Pull request by jared-esri**
@ldanzinger @GuillaumeBelz Please review. Thank you!

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+#include "ExploreScenesInFlyoverAR.h"
+
+#include "ArcGISTiledElevationSource.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "IntegratedMeshLayer.h"
+
+using namespace Esri::ArcGISRuntime;
+using namespace Esri::ArcGISRuntime::Toolkit;
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp]: totally optional, but one thing to consider could be using a namespace alias to make it very clear that some of these objects come from the toolkit. `namespace toolkit = Esri::ArcGISRuntime::Toolkit;`

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    {
+      qDebug() << e.code() << e.message() << " - " << e.additionalMessage();
+      return;
+    }
+
+    // Enable subsurface navigation. This allows you to look at the scene from below.
+    m_scene->baseSurface()->setNavigationConstraint(NavigationConstraint::None);
+
+    // Start with the camera at the center of the mesh layer.
+    m_originCamera = Camera(m_integratedMeshLayer->fullExtent().center().y(), m_integratedMeshLayer->fullExtent().center().x(), 250, 0, 90, 0);
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp]: would be cleaner to get the `m_integratedMeshLayer->fullExtent().center()` once, then access x and y in the Camera c'tor

File: ArcGISRuntimeSDKQt_QMLSamples/Scenes/ExploreScenesInFlyoverAR/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## How to use the sample
+
+When you open the sample, you'll be viewing the scene from above. You can walk around, using your device as a window into the scene. Try moving vertically to get closer to the ground.
+
+## How it works
+
+1. Create the `ArcGISARView` and add it to the view.
+2. Create the scene, add content, then display it.
+4. When the content you want to view loads, get its center point and use that to create the origin camera for the AR view. Note that the altitude should be set so that all scene content is visible. For a city, a good value might be a bit higher than the tallest building. The sample uses 250 meters in the absence of tall buildings in the sample data.
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Scenes/ExploreScenesInFlyoverAR/README.md]: numbering is off

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp [hunk 4]
```diff
@@ -0,0 +1,116 @@
+// [WriteFile Name=ExploreScenesInFlyoverAR, Category=Scenes]
+// [Legal]
+// Copyright 2019 Esri.
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp]: 2020 for new files.

Comment (DonMoJa) [reply ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp]: Thanks for catching that. New year always catches me by surprise.

Comment (DonMoJa) [reply ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp]: Need to update the samples template as well as they reference 2019 still.

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp (hunk 1 above)
Comment (DonMoJa) [reply ArcGISRuntimeSDKQt_CppSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cpp]: I like this idea. Helps to highlight the objects coming from the toolkit.

**Comment by jared-esri** [comment]
Got some feedback from the samples team review about a change to the design and also made a couple other minor changes. If you could take a quick look over them that would be great.

I know we don't do this for other samples but to access the data for this sample, the ssl.so libraries are required for android. Should this be included in the .pro files or leave this up to the user?

Thanks!

File: ArcGISRuntimeSDKQt_QMLSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.qml [hunk 5]
```diff
@@ -42,6 +42,7 @@ Rectangle {
             BasemapImagery {}
 
             Surface {
+                opacity: 0.0;
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Scenes/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.qml]: semi colon not needed for properties in QML

**Pull request merged**